docs/DeveloperGuide.md

```diff
@@ -334,6 +334,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 *{More to be added}*
 
+**Use case: Open a commission**
```

Comment:
Should we specify that a customer has to be open (e.g. `Preconditions: AB currently has a customer open.`)